package.json

```diff
@@ -2,25 +2,54 @@
     "name": "png-js",
     "description": "A PNG decoder in JavaScript",
     "version": "1.0.0",
+    "type": "module",
+    "main": "./lib/png-js.cjs",
+    "module": "./lib/png-js.js",
+    "browser": {
```

Comment:
Instead of using non standard browser field, use nested conditions in exports field:
https://nodejs.org/api/packages.html#nested-conditions

IMO, for browser should have only import (es module)

This should be future proof.

Reply:
Sgtm! Thanks for the input. Done in [decd6ba](https://github.com/foliojs/png.js/pull/81/commits/decd6ba8b3ff4488e50a4c4054770cb22fb08deb)

Reply:
I think we might still need the browser field for backward compatibility though, eg for old tools that don't support exports. Otherwise we'll have to release this in a major version.

Reply:
Done! I also wanted to remove `Buffer` usage in favor of Uint8Array (separately) if that sounds good for you guys. I think that should be breaking, but up to you 😄